### PR TITLE
chore: update app translations

### PR DESCRIPTION
**Pull request by firebomb1**
## Description
Biweekly translation sync by the MKBD Workstream.

Chinese translations which were missing from last sync are now caught up.

## Pull Request Type

- [ ] :bug: Bug fix (Non-breaking Change: Fixes an issue)
- [x] :hammer_and_wrench: Chore (Non-breaking Change: Doc updates, pkg upgrades, typos, etc..)
- [ ] :nail_care: New Feature (Breaking/Non-breaking Change)

None, only translations.

> What protocols, transaction types or contract interactions might be affected by this PR?

None.

### CODE REVIEW

File: src/assets/translations/fr/main.json [hunk 1]
```diff
@@ -426,7 +432,10 @@
         },
         "cannotDepositWhilePendingTx": "Impossible de déposer tant que des Txs sont en attente",
         "cannotWithdrawWhilePendingTx": "Impossible de retirer tant que des Txs sont en attente",
-        "dangerousWithdrawWarning": "Les frais THORChain pourraient entraîner la perte totale des fonds retirés. Envisagez de retirer un montant plus élevé pour éviter ce problème."
+        "dangerousWithdrawWarning": "Les frais THORChain pourraient entraîner la perte totale des fonds retirés. Envisagez de retirer un montant plus élevé pour éviter ce problème.",
+        "agreeRunePool": {
+          "1": "Je comprends que RUNEPool a à la fois un blocage minimum de 90 jours et un risque accru de perte impermanentes."
```
Comment (gomesalexandre) [inline src/assets/translations/fr/main.json:437]:
```suggestion
          "1": "Je comprends que RUNEPool a à la fois un blocage minimum de 90 jours et un risque accru de perte impermanente."
```

or

```suggestion
          "1": "Je comprends que RUNEPool a à la fois un blocage minimum de 90 jours et un risque accru de pertes impermanentes."
```

File: src/assets/translations/fr/main.json [hunk 2]
```diff
@@ -657,7 +666,8 @@
         "stake": "Miser",
         "withdraw": "Retirer",
         "unstakeRequest": "Demande de démise",
-        "setRuneAddress": "Définir l'adresse Rune"
+        "setRuneAddress": "Définir l'adresse Rune",
```
Comment (gomesalexandre) [inline src/assets/translations/fr/main.json:669]: That's wrong in the EN flavor of this copy but should be `RUNE` if you feel like updating it while at it 🙏🏽

**Review by gomesalexandre — APPROVED** [review]
Two smolish improvements to tackle at your convenience (i.e resolve or tackle comments), but otherwise LGTM, get in!

**Comment by firebomb1** [comment]
Thanks for the review, fixed "RUNE" in English and all the languages so we don't retranslate/pay for them again just for an uppercase issue.

File: src/assets/translations/fr/main.json [hunk 3]
```diff
@@ -915,7 +927,11 @@
     "enableMetaMaskSnap": "utilisez le Snap MultiChain",
     "connectChain": "connectez %{chainName}",
     "approvalGasFee": "Frais d'autorisation de carburant %{fee}",
+    "approvalGasFeeLoading": "Chargement des frais de carburant pour l'autorisation...",
+    "approvalResetGasFee": "Frais de carburant pour réinitialisation d'autorisation %{fee}",
+    "approvalResetGasFeeLoading": "Chargement des frais de carburant pour réinitialisation de l'autorisation...",
```
[the remark this answers is not in the document]
Comment (firebomb1) [reply src/assets/translations/fr/main.json]: Option 1 chosen. Thx!

File: src/assets/translations/fr/main.json (hunk 2 above)
Comment (firebomb1) [reply src/assets/translations/fr/main.json]: Corrected in all languages including English.

File: src/assets/translations/fr/main.json (hunk 1 above)
Comment (firebomb1) [reply src/assets/translations/fr/main.json]: Option 2 chosen. Thx!

**Review by gomesalexandre — APPROVED** [review]

**Review by 0xApotheosis — APPROVED** [review]

**Pull request merged**